File: satire/satireReal.go

package main

import (
	"fmt"
)

// This is a contant

const STARTING_X = 5
const STARTING_Y = 5
const TWO_PI     = 6.0

var radiusValue [5]int

// This is a comment
func findRadius(centerX, centerY, radius, pointX, pointY int) int {
	multiX := centerX - pointX
	multiY := centerY - pointY
	multiXY := 0
	multiXY = (multiX * multiX) + (multiY * multiY)
	if multiXY >= (radius * radius) {
		return 0
	}
	return 1
}

func areaOperator(findX, findY, loop int) float64 {
	valueOut := 0.0
	valueOut = (TWO_PI * float64(loop)) / (float64(findX*findX) + float64(findY*findY))
	return valueOut
}

func partialFind(centerX, centerY, radius int) {
	const INCREMENTER = 1
	loop := 0
	localRadius := [5]int{}
	localFloat := [5]float64{}
	for loop < radius {
		findX := centerX - loop
		findY := centerY - loop
		radiusValue[loop] = findRadius(centerX, centerY, radius, findX, findY)
		localFloat[loop] = areaOperator(findX, findY, loop)
		localRadius[loop] = radiusValue[loop] + INCREMENTER
		fmt.Println("Some command")
		fmt.Println(radiusValue[loop])
		fmt.Println(localFloat[loop])
		loop += localRadius[loop]
	}
}

func main() {
	partialFind(STARTING_X, STARTING_Y, 5)
}

File: satire/satireReal.rb
# This is a contant
STARTING_X = 5
STARTING_Y = 5
TWO_PI = 6.0

$radiusValue = Array.new(5)

# This is a comment
def find_radius(center_x, center_y, radius, point_x, point_y)
  multi_x = center_x - point_x
  multi_y = center_y - point_y
  multi_xy = (multi_x * multi_x) + (multi_y * multi_y)
  if multi_xy >= (radius * radius)
    return 0
  else
    return 1
  end
end

def area_operator(find_x, find_y, loop)
  value_out = (TWO_PI * loop) / ((find_x * find_x) + (find_y * find_y))
  return value_out
end

def partial_find(center_x, center_y, radius)
  incrementer = 1
  loop = 0
  local_radius = Array.new(5)
  local_float = Array.new(5)
  while loop < radius
    find_x = center_x - loop
    find_y = center_y - loop
    $radiusValue[loop] = find_radius(center_x, center_y, radius, find_x, find_y)
    local_float[loop] = area_operator(find_x, find_y, loop)
    local_radius[loop] = $radiusValue[loop] + incrementer
    puts "Some command"
    puts $radiusValue[loop]
    puts local_float[loop]
    loop += local_radius[loop]
  end
end

partial_find(STARTING_X, STARTING_Y, 5)

